Cargo.toml

```diff
@@ -36,6 +37,10 @@ ron = { version = "0.6", optional = true }
 json5_rs = { version = "0.3", optional = true, package = "json5" }
 
 [dev-dependencies]
-serde_derive = "1.0.8"
+chrono = {version = "0.4", features = ["serde"]}
 float-cmp = "0.8"
-chrono = { version = "0.4", features = ["serde"] }
```

Comment:
Moving the `chrono` and `serde_derive` lines is also not necessary.

Reply:
Done